doc/redirects.csv

```diff
@@ -1380,6 +1380,7 @@ generated/pandas.unique,../reference/api/pandas.unique
 generated/pandas.util.hash_array,../reference/api/pandas.util.hash_array
 generated/pandas.util.hash_pandas_object,../reference/api/pandas.util.hash_pandas_object
 generated/pandas.wide_to_long,../reference/api/pandas.wide_to_long
+reference/api/pandas.core.window.Rolling,../reference/api/pandas.core.window.rolling.Rolling.apply
```

Comment:
```suggestion
reference/api/pandas.core.window.Rolling,pandas.core.window.rolling.Rolling.apply
```

I think it is like this. But can you build the docs to check if the redirect works?

Reply:
Can you tell me how to build the docs?

Reply:
(is it running `docs/make.py` or something...)

Reply:
@rushabh-v 

Inside the `doc` folder, run
```
python make.py clean
python make.py --single <your file goes here>
```

(see [building the documentation](https://pandas.pydata.org/pandas-docs/stable/development/contributing.html#building-the-documentation))

Reply:
I ran `python make.py html` but it says,
```
Running Sphinx v2.1.2
Theme error:
no theme named 'pandas_sphinx_theme' found (missing theme.conf?)
```

Reply:
What should I do in this case?
I have made the requested change but I am able to check if it's working properly or not?

Reply:
If you create an environment according to the contributing docs, it should be included. But it was only added more recently, so if you created your environment before that, you can install it: 

https://github.com/pandas-dev/pandas/blob/fa2aa9f54dc125e4d9a3546f99ec5bf3e1b50e4d/environment.yml#L105-L107

So something like `pip install git+https://github.com/pandas-dev/pandas-sphinx-theme.git@master` should do it

Reply:
Okay now it worked.

I ran `python make.py html` for both the cases above,  And it said,
```
build succeeded, 3 warnings.

The HTML pages are in build/html.
```
both the time.

The three warnings are 
```
/home/rushabh/pandas/doc/source/whatsnew/v1.0.0.rst:1307: WARNING: Cannot find contributors for range 'v0.25.3..v1.0.0': Cmd('git') failed due to: exit code(128)
  cmdline: git shortlog -s v0.25.3..v1.0.0
  stderr: 'fatal: ambiguous argument 'v0.25.3..v1.0.0': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]''
/home/rushabh/pandas/doc/source/whatsnew/v1.0.1.rst:93: WARNING: Cannot find contributors for range 'v1.0.0..v1.0.1|HEAD': Cmd('git') failed due to: exit code(128)
  cmdline: git shortlog -s v1.0.0..HEAD
  stderr: 'fatal: ambiguous argument 'v1.0.0..HEAD': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]''
/home/rushabh/pandas/doc/source/whatsnew/v1.0.2.rst:56: WARNING: Cannot find contributors for range 'v1.0.1..v1.0.2|HEAD': Cmd('git') failed due to: exit code(128)
  cmdline: git shortlog -s v1.0.1..HEAD
  stderr: 'fatal: ambiguous argument 'v1.0.1..HEAD': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]''
```

Reply:
You can ignore those warnings.

Were you able to check if the redirect is working?

Reply:
It says build succeeded Doesn't that mean the redirect is working. Or I have to check something else also?

Reply:
No, the build does not check that. You need to actually check that page, to see if it redirects to the correct other page, by opening the built documentation in a browser.

Reply:
Open `docs/build/html/index.html` in the browser and check that the redirection is working :)

Reply:
Yes, I checked it. The redirect works fine now.

Reply:
Okay I'll be adding a commit soon.
And that review was requested by mistake.